pynessie/model.py

```diff
@@ -539,6 +539,8 @@ class DiffResponse:
     """Dataclass for a DiffResponse."""
 
     diffs: List[DiffEntry] = desert.ib(fields.List(fields.Nested(DiffEntrySchema(), data_key="diffs")))
+    has_more: bool = attr.ib(default=False, metadata=desert.metadata(fields.Bool(allow_none=True, data_key="hasMore")))
+    token: str = attr.ib(default=None, metadata=desert.metadata(fields.Str(allow_none=True)))
```

Comment:
are these specific to DiffResponse or are we missing a common base class for paged responses ?

Reply:
Never had a base-class nor "proper, generic paging support" before

Reply:
BTW: REST API v1 doesn't actually support paging except for the commit-log.